## OpenAI lab
- connect to [openai platform](https://openai.com/api/)
- login to the api platform (top right)
- Click on **getting started**
- Choose OS and click on **Create ab API key**
- A good place to store the file (Windows) would be in the $PROFILE init file
- Click on **Create new secret key**
- Make sure you copy is - then sabe it like this:  
setx OPENAI_API_KEY "your_api_key_here"
- Run (reload) it like this:  
```. $PROFILE```

Then try the following:




In [1]:
from openai import OpenAI
client = OpenAI()

response = client.responses.create(
    model="gpt-5-nano",
    input="Write a one-sentence bedtime story about a unicorn."
)

print(response.output_text)

RateLimitError: Error code: 429 - {'error': {'message': 'You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.', 'type': 'insufficient_quota', 'param': None, 'code': 'insufficient_quota'}}

This is the sign (HTTP errir 429) that you have not paid them...  
After paying:

In [2]:
from openai import OpenAI
client = OpenAI()

response = client.responses.create(
    model="gpt-5-nano",
    input="Write a one-sentence bedtime story about a unicorn."
)

print(response.output_text)

Under the silver moon, a gentle unicorn curled up beside a whispering brook and drifted into dream as the stars sang a soft lullaby.
